packages/wallets/src/components/OptionsAndMultipliersListing/OptionsAndMultipliersListing.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import IcAppstoreBinaryBot from '../../public/images/ic-appstore-binary-bot.svg';
+import IcAppstoreDerivBot from '../../public/images/ic-appstore-deriv-bot.svg';
+import IcAppstoreDerivGo from '../../public/images/ic-appstore-deriv-go.svg';
+import IcAppstoreDerivTrader from '../../public/images/ic-appstore-deriv-trader.svg';
+import IcAppstoreSmartTrader from '../../public/images/ic-appstore-smart-trader.svg';
+import { TradingAccountCard } from '..';
+import './OptionsAndMultipliersListing.scss';
+
+const options_and_multipliers = [
+    {
+        title: 'Deriv Trader',
+        description: 'Options and multipliers trading platform',
+        icon: <IcAppstoreDerivTrader />,
+    },
+    {
+        title: 'Deriv Bot',
+        description: 'Automated trading, no coding required',
+        icon: <IcAppstoreDerivBot />,
+    },
+    {
+        title: 'SmartTrader',
+        description: 'Our legacy options trading platform',
+        icon: <IcAppstoreSmartTrader />,
+    },
+    {
+        title: 'Binary Bot',
+        description: 'Our legacy automated trading platform',
+        icon: <IcAppstoreBinaryBot />,
+    },
+    {
+        title: 'Deriv GO',
+        description: 'Multipliers trading on the go',
+        icon: <IcAppstoreDerivGo />,
+    },
+];
+
+const OptionsAndMultipliersListing = () => {
+    return (
+        <div className='wallets-options-and-multipliers-listing'>
+            <section className='wallets-options-and-multipliers-listing__header'>
+                <div className='wallets-options-and-multipliers-listing__header-title'>
+                    <h1>Options & multipliers</h1>
```

Comment:
@farhan-nurzi-deriv do we have any plan for translation?

Reply:
Not yet :(, maybe after we have done the generic components we can start on translation